opentasks/src/main/res/values-de/strings.xml

```diff
@@ -52,7 +52,7 @@
     <!-- Task editor -->
     <string name="activity_edit_task_title">Bearbeiten</string>
     <string name="activity_add_task_title">Erstellen</string>
-    <string name="activity_edit_task_error_invalid_url">Ungültige URL</string>
+    <string name="activity_edit_task_error_invalid_uri">Ungültige URI</string>
```

Comment:
I don't think we should change the translations. The term "URL" probably makes more sense than "URI" to most users.

Reply:
Ok, I've reverted those changes, and squashed-rebased so that there is only one commit with the reduced changeset. Ready for check.